SchedulerJobs/SchedulerJobs.UnitTests/Services/ReconcileHearingAudioServiceTests.cs

```diff
@@ -0,0 +1,90 @@
+﻿using Microsoft.Extensions.Logging;
+using Moq;
+using NUnit.Framework;
+using SchedulerJobs.Services;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using VideoApi.Client;
+using VideoApi.Contract.Responses;
+
+namespace SchedulerJobs.UnitTests.Services
+{
+    public class ReconcileHearingAudioServiceTests
```

Comment:
Logger calls not verified

Reply:
as discussed these are not required to be verified per functionality